recipes/hdf5/all/conanfile.py

```diff
@@ -271,6 +271,7 @@ def _config_libname(lib):
         self.cpp_info.set_property("cmake_find_mode", "both")
         self.cpp_info.set_property("cmake_file_name", "HDF5")
         self.cpp_info.set_property("cmake_target_name", "HDF5::HDF5")
+        self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])
```

Comment:
Not like this please. You have to split the logic inside this file in 2 files:
- conan v1 tricks (target aliases) => to include into legacy cmake_find_package* generators
- `HDF5_IS_PARALLEL` variable => to include into legacy cmake_find_package* generators AND cmake_build_modules property (for CMakeDeps).

Reply:
ok, how about this next commit?